include/unifex/receiver_concepts.hpp

```diff
@@ -211,6 +190,28 @@ inline constexpr bool is_receiver_query_cpo_v = !is_one_of_v<
 template <typename T>
 using is_receiver_cpo = std::bool_constant<is_receiver_cpo_v<T>>;
 
+#if UNIFEX_CPP_CONCEPTS
```

Comment:
is this supposed to be UNIFEX_CXX_CONCEPTS?

Reply:
Yep.